ajax/libs/noti.js/package.json

```diff
@@ -16,6 +16,6 @@
   "license": "GPL-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/j-l-n/noti.js.git"
+    "url": "https://github.com/needim/noty.git"
```

Comment:
@sufuf3 Not sure
The earliest time stamp of `0.9-beta.5` In our repo is [17/10/2014](https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/noti.js/0.9-beta.5/noti.js#L7)
But in needim, `2.2.9` released in [26/9/2014](https://github.com/needim/noty/commits/d0ad9b68bbccd250b25f740e1684430967eb5876)

Also j-l-n said at [2/11/2014](https://github.com/cdnjs/cdnjs/pull/3983#issuecomment-61408355)
> Just to clarify once again why this trouble has arised: I removed the entire old repository (beta version) and then started with a new, clean Commit (v1.0.0).

So I think needim/noty is **not** suitable for substituting `j-l-n/noti.js` to `needim/noty`.